migrations/migrations/0016_remove_users_pdf_url.py

```diff
@@ -0,0 +1,13 @@
+from datastore.migrations import RemoveFieldMigration
```

Comment:
merge into one migration for less overhead

Reply:
Why that? The two migration make their work and are configured default migrations.

Reply:
Two reasons:
1) Each migration produces additional overhead. Going once through thousands of positions and applying some changes is more performant than doing the same thing twice.
2) Same or similar changes should be gathered in one migration to for easier overview.
The latter is discussable since "remove field x" might also be seen as a simple atomic change, but the more important reason IMO is 1), anyway.

Reply:
Okay, write a merged migration and updated index.